packages/calcite-components/src/components/chip-group/chip-group.tsx

```diff
@@ -176,6 +168,23 @@ export class ChipGroup implements InteractiveComponent {
     }
   }
 
+  @Listen("calciteInternalChipSelect")
+  calciteInternalChipSelectListener(event: CustomEvent): void {
+    if (event.composedPath().includes(this.el)) {
```

Comment:
[Per our guidelines](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/README.md#privateinternal-events), custom events should be stopped after handling. Sorry for not catching this earlier. 😅

Applies to other custom event handlers.